### DISCUSSION TITLE
Missing icons in Airflow website for Firefox

### DISCUSSION THREAD

**Discussion by enricogandini** [apache/airflow#25451] [General]
### Apache Airflow version

2.3.3 (latest released)

### What happened

I just started using Airflow. After initializing the airflow database and creating a user, I initialized the Airflow webserver and scheduler.

Using the Firefox browser, I went to localhost:8080, and I logged in. Many icons were missing, as you can see from the screenshot
![Screenshot 2022-08-01 at 14-19-01 DAGs - Airflow](https://user-images.githubusercontent.com/52758483/182147575-ea9ba442-1c88-44ba-b568-0c4fd331ef1d.png)
.

I am running Airflow on the Ubuntu 20.04 operating system, with Firefox 103.0.
I installed Airflow 2.3.3 through Anaconda (using the conda-forge channel). I also tried reinstalling Airflow using pip, but the problem persisted.

I then tried opening the Airflow website on Chrome: everything was fine, all the icons were shown properly.

### What you think should happen instead

The icons should have been properly displayed.

### How to reproduce

airflow webserver
airflow scheduler
Go to: localhost:8080, and login.
You will notice that the icons are not displayed!

### Operating System

Ubuntu 20.04.4 LTS

### Deployment

Virtualenv installation

### Deployment details

I installed Airflow 2.3.3 with Anaconda 4.13.0, through the conda-forge channel.
Even after reinstalling with pip, the problem persisted.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
First of all - make sure you install airflow with the only supported solution which is `pip` and constraints: https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html. - you might have some bad dependencies if you don't.

Secondly please look at the output of errrors you have in developer tools of Firefox - without it it is impossible to help you and it is likely when you look at the output you will find out the root cause of the problem - most likely something is blocking your requests - but we will find out when you provide more logs from firefox 

See https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/ and related tools. Any errors that you will see there would be useful.

**Comment by enricogandini** [comment]
Thank you for your answer, @potiuk !

I reinstalled Airflow with pip. The problem persists when using Firefox.

When navigating the Airflow website with the Firefox Network Monitor open, I don't see any errors. I either get HTTP status 200 or 304, but there aren't any HTTP error status codes (4XX or 5XX).
The only picture that I retrieve correctly is `pin_32.png` (the Airflow icon): I get it with a status code 200.

I do not know what other requests are supposed to retrieve the missing icons (for instance, `play_arrow`, `delete_outline`, `unfold_more`...).

I will copy-paste here the URLs of a couple of responses that may be interesting: the URLs to the CSS files of Font Awesom and Material Icons. Both of them are status codes 304.
```
http://localhost:8080/static/appbuilder/css/font-awesome.min.css
http://localhost:8080/static/dist/materialIcons.b5be025c4c658382069b.css
```

Of course I tried refreshing the page, and also opening the page on a Firefox private window, to no avail: the missing icons keep being missing!
On Chrome, everything works just fine.

**Comment by potiuk** [comment]
Can't help you unless you track it down. This must be somethign with your firefox because my firefox looks like this:

![image](https://user-images.githubusercontent.com/595491/182395009-17d78aaf-b992-4ba3-b94b-11c559d3d41f.png)

so this is "can't reproduce" - unless you can provide more information or do more checks, there is not much I or anyone else can do

**Reply by potiuk** [reply]
Clear your cache - reinstall firefox, run it in private window, install firefox on a different machines - all this might give you clues, but you need to continue digging yourself.
